fpdf/fpdf.py

```diff
@@ -279,7 +279,9 @@ def __init__(
         but is less compatible with the PDF spec.
         """
         self.page = 0  # current page number
-        self.pages = {}  # array of PDFPage objects starting at index 1
+        self.pages: Dict[int, PDFPage] = (
+            {}
+        )  # array of PDFPage objects starting at index 1
```

Comment:
```suggestion
        # array of PDFPage objects starting at index 1:
        self.pages: Dict[int, PDFPage] = {}
```
Just a suggestion to make the code clearer